test/unit/org/apache/cassandra/cql3/ast/CreateIndexDDL.java

```diff
@@ -55,6 +61,10 @@ default UnquotedSymbol longName()
             return name();
         }
         boolean supported(TableMetadata table, ColumnMetadata column);
+        default EnumSet<QueryType> supportedQueries(AbstractType<?> type)
```

Comment:
this is needed by the model to know what operations could be done by the index.  maintaining this is actually kind of a burden, so would be good to figure out how to ask the real impl....